test/Gruntfile.js

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+module.exports = function(grunt) {
```

Comment:
Do we actually need Grunt anymore? For example, `mocha` has watch behavior by default now.

Reply:
That's good news that I was unaware of.  We don't want to get sidetracked with that right now though.  Create an issue to implement that later if you like.

Reply:
np